recipes/pycares/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "pycares" %}
+{% set version = "3.0.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: b253f5dcaa0ac7076b79388a3ac80dd8f3bd979108f813baade40d3a9b8bf0bd
+
+build:
+  number: 0
+  skip: True  # [py<=35 or win]
```

Comment:
Do you need to skip windows? Upstream claims it should work

Reply:
I tried building it for Windows as well, but it failed for Windows with the following error:

[Link to error](https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=80399&view=logs&jobId=240f1fee-52bc-5498-a14a-8361bde76ba0&taskId=23010c01-6a9d-5f94-9037-792618644b77&lineStart=7409&lineEnd=7410&colStart=1&colEnd=1)

```
conda_build.exceptions.DependencyNeedsBuildingError: Unsatisfiable dependencies for platform win-64: {'colcon-output', 'colcon-recursive-crawl', 'colcon-metadata', 'colcon-notification', 'colcon-parallel-executor', 'colcon-devtools', 'colcon-ros', 'colcon-cmake'
```

I check a couple of these packages, and they seem to exists as `noarch`, so I'm not sure what the issue is.  I'll try the build again including Windows and see if it's still a problem.